src/interface.tsx

```diff
@@ -4,6 +4,11 @@ export interface AppState {
   isDarkModeOn: boolean;
 }
 
+export interface NavProps {
+  darkMode: string | null;
+  toggleDarkMode: () => void;
+}
+
```

Comment:
Thank you for moving this into the interface file! Way to keep the structure of our files more modular!